README.md

```diff
@@ -565,37 +632,78 @@ class Student {
 This `Student` class has several attributes, as we've seen before with
 `interfaces`, but it also has:
 
-1. A constructor: this is a special method that gets called when an object of
+1. A `constructor`: this is a special method that gets called when an object of
    this type is created. The constructor has the responsibility to initialize
-   the object with its proper values, sometimes based on defaults, sometimes
-   based on values passed into it.
-2. 2 functions that can be called on any object of this type. These functions
-   can the implement functionality that will be available and standard for all
-   objects of this type.
-3. A private member variable: this member variable, in this case `courses` will
-   not be accessible outside the code in this class. This allows us to have
+   the object with its proper values, sometimes based on defaults like `role` in
+   our example, and sometimes based on values passed into it.
+2. Two functions that can be called on any object of this type. These functions
+   can then implement functionality that will be available and standard for all
+   objects instantiated from this class.
+3. A `private` member variable. Private member variables, in this case `courses`, 
+   are not accessible outside the code in this class. This allows us to have
    "internal" state that can be changed without affecting the users of this
    class.
 
-TypeScript gives us a shorthand for defining properties, including private ones,
-using the constructor without having to define the member variables separately.
-The following code is functionally exactly equivalent to the code above:
+You may notice that our example seems to have a lot of redundant code. First, we 
+define all the properties and their types we expect our class to have. Then, we define
+those same properties and types in the parameter of the `constructor`. Finally, we 
+define the actual properties themselves based off the parameters and defaults. 
+
+Thankfully, TypeScript gives us a shorthand for defining properties and their types, 
+including private members, by just using the constructor. The following code is functionally
+equivalent to the code above:
```

Comment:
Might also want to include an example of how to instantiate an instance of the class